utils/sysfs/sysfs.go

```diff
@@ -378,16 +389,20 @@ func getCPUID(dir string) (uint16, error) {
 	return 0, fmt.Errorf("can't get CPU ID from %s", dir)
 }
 
+// cpuList allows to determine quickly if CPU or particular ID is online.
 type cpuList struct {
-	online          map[uint16]interface{}
+	// online maps CPU ID to a struct for each online CPU.
+	online map[uint16]struct{}
+	// hotplugDisabled is true when kernel CPU hotplug is disable and all CPUs are assumed online.
 	hotplugDisabled bool
```

Comment:
thanks for adding the hot plug check upfront in `IsCPUOnline`. Now that we have that check can we remove this and the duplicate check in `newCPUList`?